### ISSUE TITLE
tell plugin devs to update their projects and review changes

### ISSUE THREAD

**Issue by nixjdm** [lektor/lektor-website#196]
I do not think #194 needs to wait on this, but we should tell all of the plugin devs that this change has been made. They may want to edit their category or tags that I chose for them. Many of them also do not use all of the fields we pull from PyPI, so they can update their plugins.

**Comment by goanpeca** [comment]
What do you think of creating a team inside the lektor org for all the plugins developers?
so that `@team-plugin-devs` or similar can ping all of them  ?

**Comment by nixjdm** [comment]
Sounds good!

**Comment by nixjdm** [comment]
@lektor/plugin-developers test ping!

**Comment by goanpeca** [comment]
Works as expected :-)

**Comment by nixjdm** [comment]
I just made #203, a new issue to keep things tidy, and invited everyone to the plugin-developers team. @goanpeca thanks for the tests!
